routes/blogy.js

```diff
@@ -43,26 +42,29 @@ router.get("/blog/:id/blogcomment", async (req, res) => {
 
 
 // To post comments to blog
-router.post("/blogcomment", async (req, res) => {
-  const { title, content } = req.body;
-  const postComment = new blogcomment({ title, content });
-
+router.post("/comments", async (req, res) => {
+  const { comment, content } = req.body;
+  const postComment = new blogcomment({ comment, content });
   // save the comment to database
   try {
       await postComment.save()
       res.send("Comment Sent");
```

Comment:
It might be better to send the comment that was just created instead of sending a text

Reply:
oh ok. I get this. I will change it.